Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml

```diff
@@ -24,21 +24,21 @@
             {
                 <div class="govuk-summary-list__row">
                     <dt id="edubase-trigger-1" class="govuk-summary-list__key">EduBase trigger 1</dt>
-                    <dd class="govuk-summary-list__value">@Model.Establishment.HelpdeskTrigger1</dd>
+                    <dd id="edubase-trigger" class="govuk-summary-list__value">@Model.Establishment.HelpdeskTrigger1</dd>
                 </div>
             }
             @if (Model.DisplayPolicy.HelpdeskPreviousLocalAuthorityId)
             {
                 <div class="govuk-summary-list__row">
                     <dt id="previous-local-authority" class="govuk-summary-list__key">Previous local authority</dt>
-                    <dd class="govuk-summary-list__value">@Model.HelpdeskPreviousLocalAuthorityName</dd>
+                    <dd id="previous-local-authority-info" class="govuk-summary-list__value">@Model.HelpdeskPreviousLocalAuthorityName</dd>
                 </div>
             }
             @if (Model.DisplayPolicy.HelpdeskPreviousEstablishmentNumber)
             {
                 <div class="govuk-summary-list__row">
                     <dt id="previous-establishment-number" class="govuk-summary-list__key">Previous establishment number</dt>
-                    <dd class="govuk-summary-list__value">@Model.Establishment.HelpdeskPreviousEstablishmentNumber</dd>
+                    <dd id="previous-establishment-number-info" class="govuk-summary-list__value">@Model.Establishment.HelpdeskPreviousEstablishmentNumber</dd>
```

Comment:
What is the `-info` part for? I'm assuming you're going to assert the text value in this element so probably worth swapping out `-info` for `-value` is that's the case.

Reply:
will update and push the code